lib/simple_form/components/input_groups.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+module SimpleForm
+  module Components
+    module InputGroups
+      module Prepends
+        def prepend(wrapper_options = nil)
+          template.content_tag(:div, content_tag(:span, options[:prepend], class: "input-group-text"), class: "input-group-prepend")
+        end
+      end
+
+      module Appends
+        def append(wrapper_options = nil)
+          template.content_tag(:div, content_tag(:span, options[:append], class: "input-group-text"), class: "input-group-append")
```

Comment:
To improve the readability, we can extract the span tag, like this:

```ruby
span_tag = content_tag(:span, options[:append], class: "input-group-text")
template.content_tag(:div, span_tag, class: "input-group-append")
```

Wdyt?